### DISCUSSION TITLE
Any help needed? Where should I start ?

### DISCUSSION THREAD

**Discussion by jarlah** [jdbi/jdbi#2334] [General]
I worked on the freemarker plugin way back. Not the biggest contribution in the world, but hey it worked and it was even used extensively in a real production project (where it is now however I cannot say 😆). A million years later (ok, 5 years..) I'm open for new adventures ;) Anything low hanging or simple to pick on ? Or is my help needed somewhere else ?  I still see the need for a simple jdbc layer like jdbi :) 

I see you have doc for Spring5, but for me setting up xml and all that is a :sadpanda:, so maybe add some documentation/example in the official documentation on how to integrate it with spring boot ? Because that was what I was doing with jdbi five years ago. And I bet there are more idiomatic ways to do it now (non-rhetorical question) ?

**Comment by stevenschlansker** [comment]
Hi @jarlah , thanks for your interest! Indeed we do not explain Spring XML configuration completely, so if that is something you use and would recommend to others, adding documentation could make sense.
Aside from that, I would recommend taking a look through the issues list and see if any features or bugs catch your eye. If you see one you would like to work on and would like guidance, please feel free to ask what direction to go in :)

**Reply by jarlah** [reply]
probably looking into documenting adding jdbi to a normal spring boot app. I think that's the most useful aspect for using jdbi as far as I am concerned atm.

**Reply by jarlah** [reply]
@stevenschlansker I think we should make an examples folder with actual examples and not just code. When I see an examples folder I expect to see runnable standalone examples with their own poms etc. 

I have made a simple spring boot project [here](https://github.com/jarlah/jdbi-spring-boot-demo) that shows a simple Spring Boot app configured with TransactionAwareDataSourceProxy. Im pretty sure this is the setup I used back in the days.

But, if we are going to add such a project into an examples folder, we should endorse the pattern. So any code in that demo project should be jdbi best practices, if there such a thing ;)

**Reply by stevenschlansker** [reply]
@jarlah Sure, we'd be happy to start collecting examples. Would you be interested in contributing the example in that repo, maybe with a fleshed out test case that shows it works, and we can add a link to it from the docs?

**Comment by stevenschlansker** [comment]
PS - we use the freemarker plugin extensively, so thank you for that!
